tdrs-frontend/src/App.js

```diff
@@ -17,7 +17,15 @@ import Footer from './components/Footer'
 function App() {
   return (
     <>
-      <a className="usa-skipnav" href="#main-content">
+      <a
+        className="usa-skipnav"
+        href="#main-content"
+        onKeyPress={(e) => {
+          if (e.charCode === 32) {
+            window.location.href = '#main-content'
+          }
+        }}
+      >
```

Comment:
For some reason the `Skip Links` link was not working when the space bar was pressed on the link. This code fixes that issue.